spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -11,6 +11,9 @@
 
       result = { sp_initiated: false, oidc: false, saml_request_valid: true }
       expect(@analytics).to receive(:track_event).with('Logout Initiated', hash_including(result))
+      expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:logout_initiated).with(
```

Comment:
should/could we rebase this to use the `stub_irs_attempts_api`?

Reply:
Done! 👍